src/coprocessor/codec/convert.rs

```diff
@@ -417,7 +417,7 @@ fn round_int_str(num_next_dot: char, s: &str) -> Cow<'_, str> {
         return Cow::Borrowed(s);
     }
 
-    let mut int_str = String::new();
+    let mut int_str = String::with_capacity(s.len());
```

Comment:
It seems that a String contruct use `with_capacity(the_size)` will has `capacity` exactly equal to `the_size`, so I think `with_capacity(s.len()+1)` will better than `with_capacity(s.len())`.

Reply:
The string size will equal to `s.len()` exactly and not grow in this function, so we don't need `s.len()+1`.

Reply:
```rust
    let a = round_int_str('9', "9");
    println!("{:#?}", a);
```
output is `"10"`

Reply:
Nice catch. 👍